tests/CMakeLists.txt

```diff
@@ -25,6 +25,16 @@ file(GLOB APP_SOURCES "app/*.cpp")
 add_executable( app_test ${APP_SOURCES} )
 target_link_libraries( app_test graphene_app graphene_account_history graphene_bookie graphene_net graphene_chain graphene_egenesis_none fc ${PLATFORM_SPECIFIC_LIBS} )
 
+file(GLOB CLI_SOURCES "cli/*.cpp")
+add_executable( cli_test ${CLI_SOURCES} )
+if(WIN32)
+   list(APPEND PLATFORM_SPECIFIC_LIBS ws2_32)
```

Comment:
@srpatel19590  "PLATFORM_SPECIFIC_LIBS ws2_32" -- what are these libraries ?

Reply:
These are windows socket lib. Additionally added to PLATFORM_SPECIFIC_LIBS when the target is windows.

Reply:
@srpatel19590  - provide specific details. ie link, versions etc.

Reply:
@bobinson By compiling with Visual Studio 2017 and 2015 Update 1, the required ws2_32 lib will already available and we just required to link it. So MSVS 2017 and 2015 Update 1 is fine.

Reply:
We can remove this part if we are not using WIN32. Moreover we can use below instruction to compile in windows.
https://github.com/bitshares/bitshares-core/wiki/BUILD_WIN32#windows-development-environment-setup-and-build-instructions